app/models/comment.rb

```diff
@@ -2,6 +2,8 @@ class Comment < ApplicationRecord
   belongs_to :post, foreign_key: :post_id
   belongs_to :user, foreign_key: :user_id
 
+  after_save :update_comments_counter
+
   def update_comments_counter
     post.update(commentsCounter, post.comments.count)
   end
```

Comment:
- [ ] Line 8 in the code excerpt above has a syntax error that prevents one from creating a comment. The `after_save` callback attempts to update the post's commentsCounter when a comment is created but because of the syntax error, the entire operation fails.
![image](https://github.com/marvin-nyalik/Blog/assets/46839548/9f3bd454-518b-4d4c-a48b-be84bf6f7720)


**NB:** Kindly fix it as follows 👇 
```ruby
post.update(commentsCounter: post.comments.count)
```